services/github/github-milestone-detail.tester.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const {
+  isMetric,
+  isMetricOverMetric,
+  isIntegerPercentage,
+} = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('Milestone Open Issues')
+  .get('/issues-open/MacroPower/milestone-test/1.json')
+  .expectBadge({
+    label: '1 open issues',
```

Comment:
I find these examples slightly confusing as the titles of the milestones in your test repository are single digits. Can they be renamed to something more explicit?

Reply:
Done, the labels have now changed. Links and json files still use the ID, however.